de-webapp/src/main/java/org/iplantc/de/server/websocket/amqp/AMQPConnectionManager.java

```diff
@@ -46,6 +50,7 @@ private AMQPConnectionManager() {
             LOG.error("Exception when creating AMQP connection!" + e.getMessage());
         }
 
+        return connection;
     }
```

Comment:
The assignment to the `connection` property happens both inside `createConnection` and in all of the places where `createConnection` is called. It might be useful to remove the assignments either from the places where `createConnection` is called or from `createConnection` itself.

Reply:
Ah, good catch!